MAUI/Maps/selection.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Shape Selection in MAUI Maps widget | Syncfusion
+description: Learn here all about the Shape Selection feature of Syncfusion MAUI Maps (SfMaps) widget and more.
+platform: MAUI
+control: SfMaps
+documentation: ug
+---
+
+# Shape Selection in MAUI Maps (SfMaps)
+
+You can select a shape in order to highlight that area on a map. You can use the callback for performing any action during shape selection.
+
+## Enable shape selection
+
+You can enable shape selection on a map using the [`MapShapeLayer.EnableSelection`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeLayer/onSelectionChanged.html) property.Selection allows you select only one shape at a time. You can select a shape by tapping it. By default, the selection is enabled when the EnableSelection property is set to true.
+
+The [`ShapeSelected`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeLayer/onSelectionChanged.html) callback is used to pass the index of the selected shape when the user is selecting a shape by tapping or clicking or by programmatically.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <map:SfMaps>
+        <map:SfMaps.Layer>
+            <map:MapShapeLayer  x:Name="layer" ShapesSource="https://cdn.syncfusion.com/maps/map-data/australia.json"  ShapeSelected="MapShapeLayer_ShapeSelected"  EnableSelection="True">
+            
+            </map:MapShapeLayer>
+        </map:SfMaps.Layer>
+    </map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+private void MapShapeLayer_ShapeSelected(object sender, ShapeSelectedEventArgs e)
+{
+     layer.SelectedShapeFill = Colors.Aqua;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Enable shape selection](images/selection/enable-shape-selection.png)
+
+## Appearance customization
+
+You can customize the below appearance of the selected shape.
+
+* **SelectedShapeFill** - Change the background color of the selected shape using the [`MapShapeLayer.SelectedShapeFill`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapSelectionSettings/color.html) property.
```

Comment:
If SelectedShapeFill is null then the saturated color of the shape will be applied.
If SelectedShapeFill is Transparent, then the UI changes will not occur.